### ISSUE TITLE
CR129 - `On-device site data` and `V8 optimizer` content settings icon is inherited from Chrome 

### ISSUE THREAD

**Issue by GeetaSarvadnya** [brave/brave-browser#40993]
### Description

Found while testing https://github.com/brave/brave-browser/issues/40204
CR129 - `On-device site data` and `V8 optimizer` content settings icon is inherited from Chrome 

### Steps to reproduce

1. Clean profile `1.71.81 cr129`
2. Open brave://settings/content
3. `On-device site data` and `V8 optimizer` content settings icon is inherited from Chrome 

### Actual result

`On-device site data` and `V8 optimizer` content settings icon is inherited from Chrome 
![image](https://github.com/user-attachments/assets/72dc40ac-5804-475a-92a3-c9fc4d9f4a58)

### Expected result

The icons should be same as below:

![image](https://github.com/user-attachments/assets/1f3f94fe-f0f8-4d06-8f79-c5e5f777810f)

### Reproduces how often

Easily reproduced

### Brave version (brave://version info)

Brave | 1.71.81 Chromium: 129.0.6668.31 (Official Build) nightly (64-bit)
-- | --
Revision | 33ed75506c3a0d5a964e5ca61494d6144c52fd7d
OS | Windows 10 Version 22H2 (Build 19045.4780)

### Channel information

- [ ] release (stable)
- [ ] beta
- [X] nightly

### Reproducibility

- [ ] with Brave Shields disabled
- [ ] with Brave Rewards disabled
- [ ] in the latest version of Chrome

### Miscellaneous information

cc: @brave/qa-team @rebron @mkarolin @simonhong

**Comment by fallaciousreasoning** [comment]
@GeetaSarvadnya @simonhong I noticed upstream has actually changed nearly all the icons in the additional privacy settings. I'm going to go over them in a bit more detail

**Comment by aguscruiz** [comment]
Already resolved! :D

**Comment by rebron** [comment]
@aguscruiz @fallaciousreasoning Still an with these two icons in 1.73.x and higher
Was this pr https://github.com/brave/brave-core/pull/25524 supposed to close this.

**Comment by fallaciousreasoning** [comment]
Nah, this isn't resolved yet - @aguscruiz did decide we shouldn't replace the `V8` icon, as that's a logo

**Comment by aguscruiz** [comment]
Yeah, that V8 logo should remain as the original one, but the other icon we do have a nala version called `database`. Is there anything else needed?
